modules/appeals_api/app/serializers/appeals_api/v1/evidence_submission_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'fast_jsonapi'
+
+module AppealsApi
+  module V1
```

Comment:
Where is this being used? I see that `modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb` is calling `AppealsApi::EvidenceSubmissionSerializer.new(submissions)`. Should it be calling the V1 version instead?

But if that's the case then why is the non-V1 around? One of these serializers seems to be unused 🤔

Reply:
Ah, I think that was supposed to be removed. Must've missed it! Good catch.